include/bbp/sonata/output.h

```diff
@@ -0,0 +1,135 @@
+#pragma once
+
+#include <algorithm>
+#include <map>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include <highfive/H5File.hpp>
+
+#include <bbp/sonata/population.h>
+
+namespace H5 = HighFive;
+
+namespace bbp {
+namespace sonata {
+
+struct SONATA_API DataFrame {
+    using DataType = std::map<NodeID, std::vector<float>>;
+    DataType data;
+    std::vector<float> index;
```

Comment:
I don't like index as a name, I found it confusing. Maybe current_step? index_tstep? tstep_index?
And it should probably be double (like tstart, tstop, tstep).

Reply:
This is the name of pandas.DataFrame, I only match it.

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html

```python
dataframe = pandas.DataFrame(data=sonata.data, index=sonata.index)
```

Reply:
ok for double. I was confuse because data are in float but times are double.